integration/create_test.go

```diff
@@ -202,3 +203,32 @@ func TestSociCreate(t *testing.T) {
 		})
 	}
 }
+
+// TestSociCreateGarbageCollection ensures that SOCI index artifacts are not garbage collected after creation.
+func TestSociCreateGarbageCollection(t *testing.T) {
```

Comment:
Does this fail consistently without the changes? I wonder if this test passing isn't a strong indicator that the fix is working.

Reply:
It doesn't, and I really don't like that it doesn't.

This test more for a failsafe in case we move around some function calls and accidentally forget to label some artifacts and never notice because we don't test much in the containerd content store. I don't think there's a good way to test this fix works, though. The only way we can make sure it doesn't fail is if we call private commands in a unit test, but then of course if you call garbage collection on unlabeled zTOCs it will get deleted, so it seemed like a silly idea. This was just the best I could think of that was adjacently related.